### PR TITLE
VB-2584/VB-2588 Basic session template update

### PR DESCRIPTION
**Pull request by hutcheonb-moj**
## Description
Update session templates
Basic route/data/service level tests

Also includes the use of validationMessages which are sent from the backend when certain validation rules on changes aren't met

### CODE REVIEW

File: server/views/pages/prisons/sessionTemplates/editSingleSessionTemplate.njk [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+          },
+          classes: "govuk-!-width-one-third",
+          id: "visitRoom",
+          name: "visitRoom",
+          value: formValues.visitRoom,
+          autocomplete: "off",
+          errorMessage: errors | findError('visitRoom')
+        }) }}
+
+        {# <h2 class="govuk-heading-m">Category groups</h2>
```
Comment (psoleckimoj) [inline server/views/pages/prisons/sessionTemplates/editSingleSessionTemplate.njk:227]: Does this block want to be here if it's commented out?

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: server/views/pages/prisons/sessionTemplates/editSingleSessionTemplate.njk (hunk 1 above)
Comment (hutcheonb-moj) [reply server/views/pages/prisons/sessionTemplates/editSingleSessionTemplate.njk:227]: made the decision to leave it in as spent a fair bit of time working the logic out between here and the route file - only for us to decide not to use it, so will mean we are more prepared when we do look to include this

Comment (AledWEvans) [reply server/views/pages/prisons/sessionTemplates/editSingleSessionTemplate.njk:227]: This will be put in shortly so all good to leave for now.

**Review by AledWEvans — APPROVED** [review]
LGTM

**Pull request merged**